solution/dev_parallel/server.py

```diff
@@ -0,0 +1,54 @@
+import tornado.ioloop
+import tornado.web
+import tornado.gen
+import json
+import redis
+import uuid
+import logging
+
+from responder import result_listener
+
+# Configure logging
+logging.basicConfig(level=logging.DEBUG)
+
+r = redis.Redis(host='redis', port=6379, db=0)
```

Comment:
Hey @vzip, thank you for your interest in this challenge and submitting the solution. Unfortunately, we cant launch docker-compose based infrastructures on our side. But luckily, we have our own redis instance in cluster -- could you please make redis connection arguments "env-based" (here and everywhere redis connection is defined). i.e 
```python
from os import environ

REDIS_PASSWORD = environ.get("REDIS_PASSWORD")
REDIS_HOST = environ.get("REDIS_HOST")

# only host and pass are needed
r = redis.Redis(
        host=REDIS_HOST,
        password=REDIS_PASSWORD,
    )
```

So that we can plug correct env in and launch the solution "as intended". Thank you!

Reply:
Hey. Thank you so much for your comment. I will for shure do like you said, just few hours need to sleep first.

Reply:
changed